hledger-lib/Hledger/Read/TimeclockReader.hs

```diff
@@ -14,7 +14,7 @@ From timeclock.el 2.6:
 A timeclock contains data in the form of a single entry per line.
 Each entry has the form:
 
-  CODE YYYY/MM/DD HH:MM:SS [COMMENT]
+  CODE YYYY-MM-DD HH:MM:SS [COMMENT]
```

Comment:
https://www.ledger-cli.org/3.0/doc/ledger3.html#Time-Keeping says it contains slashes, but Ledger reads slash or dash or dot, like hledger. Emacs' timeclock.el requires slashes. I tweak this comment later maybe.

Reply:
I reverted this change for now, to keep the compatibility to the original doc and Emacs.